packages/web-twig/src/Resources/components/Header/README.md

```diff
@@ -112,10 +112,9 @@ just branding.
 | `color`    | [`transparent` \| `inverted`] | `transparent` | ✕        | Color variant                       |
 | `isSimple` | `bool`                        | `false`       | ✕        | Shorter, centered version of Header |
 
-On top of the API options, you can add `data-*` or `aria-*` attributes to
-further extend component's descriptiveness and accessibility. Also, UNSAFE styling props are available,
-see the [Escape hatches][escape-hatches] section in README to learn how and when to use them.
-These attributes will be passed to the topmost HTML element of the component.
+On top of the API options, the components accept [additional attributes][readme-additional-attributes].
+If you need more control over the styling of a component, you can use [styling props][readme-styling-props]
+and [escape hatches][readme-escape-hatches].
```

Comment:
Maybe "styling escape hatches".

Reply:
Shouldn't we change the title in the docs too then?

Reply:
It's open for a future introduction of other escape hatches, isn't it?

Reply:
we agreed on 
> you can use [style props][readme-style-props]
and [escape hatches][readme-escape-hatches].